### PR TITLE
DM-1849 Document new flags introduced for LWM2M to disable default behaviors

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
Documentation for two flags:
- disableInternalObjectInstanceActions
- disableInternalObjectInstanceActions

### CODE REVIEW

**Review by sourajitsaha17 — APPROVED** [review]

File: content/protocol-integration/lwm2m-bundle/device-details.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ The **LWM2M configuration** tab displays all LWM2M settings related to the devic
   * Binary format
   * Timestamp resources
   * Objects tab behavior (keep old values)
+  * Disable default behavior for object instances of objects 3 (Device), 4 (Connectivity Monitoring ) and 6 (Location)
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/device-details.md:70]:
```suggestion
  * Disable the default behavior for object instances of objects 3 (device), 4 (connectivity monitoring) and 6 (location)
```

File: content/protocol-integration/lwm2m-bundle/registering-lwm2m-devices.md [hunk 2]
```diff
@@ -412,6 +412,13 @@ The value must not exceed the maximum request timeout limit given in the LWM2M m
 <td style="text-align: left">Indicates the preferred content format for {{< product-c8y-iot >}} to communicate with the device. The allowed content formats are: TLV, JSON, CBOR, TEXT, OPAQUE or SENML_JSON and SENML_CBOR. In case of an empty or invalid entry, {{< product-c8y-iot >}} automatically selects the serialization format which the device sends during device registration.</td>
 <td style="text-align: left">Optional</td>
 </tr>
+<tr>
+<td style="text-align: left">Disable default behavior for object instances</td>
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/registering-lwm2m-devices.md:416]:
```suggestion
<td style="text-align: left">Disable the default behavior for object instances</td>
```

File: content/protocol-integration/lwm2m-bundle/registering-lwm2m-devices.md [hunk 3]
```diff
@@ -412,6 +412,13 @@ The value must not exceed the maximum request timeout limit given in the LWM2M m
...
+<td style="text-align: left">disableInternalObjectInstanceActions</td>
+<td style="text-align: left">Boolean</td>
+<td style="text-align: left">Cumulocity implements default handlers for Objects 3,4 and 6. For example, they update the device name upon the reception of the corresponding resource in Object 3 or update the device location. This flag allows these behaviors to be turned off.</td>
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/registering-lwm2m-devices.md:419]:
```suggestion
<td style="text-align: left">Cumulocity IoT implements default handlers for objects 3,4 and 6. For example, they update the device name upon the reception of the corresponding resource in object 3 or update the device location. This flag allows those behaviors to be turned off.</td>
```

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
what are the target versions? @sag-eweingaertner

**Review by Oezge-Akin — APPROVED** [review]

**Review by bggov — APPROVED** [review]

**Comment by sag-eweingaertner** [comment]
> what are the target versions? @sag-eweingaertner

Just 2024 (develop)

**Pull request merged**